eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -105,11 +105,11 @@ treasury_withdrawals_action = (2, { $reward_account => coin })
 
 no_confidence = (3, gov_action_id / null)
 
-new_committee = (4, gov_action_id / null, set<$committee_coldkey_credential>, committee)
+new_committee = (4, gov_action_id / null, set<$committee_cold_credential>, committee)
```

Comment:
Not specific to this change, but just something I wanted to ask in the past: 

We seem to be using the `$` prefix in a sort of ad-hoc way in  our cddl no?
In the spec it says 
`Per convention, CDDL extension points are marked with a leading
   dollar sign (types)`

Is there some rationale for it that I am not seeing?

Reply:
Honestly, I have no idea. It'd be awesome if you could research what is the correct way that we should be using :grin: